salt/metalk8s/metalk8s-ui/files/metalk8s-ui_deployment.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    k8s-app: ui
+    kubernetes.io/name: "MetalK8sUI"
+spec:
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 1
+  selector:
+    matchLabels:
+      k8s-app: ui
+  template:
+    metadata:
+      labels:
+        k8s-app: ui
+    spec:
+      tolerations:
+        - key: "node-role.kubernetes.io/master"
+          operator: "Exists"
+          effect: "NoSchedule"
+      nodeSelector:
+        beta.kubernetes.io/os: linux
+      containers:
+      - name: metalk8s-ui
+        image: localhost:5000/metalk8s-2.0/metalk8s-ui:0.2
+        imagePullPolicy: IfNotPresent
+        resources:
+          limits:
+            memory: 500Mi
+          requests:
+            cpu: 100m
+            memory: 70Mi
```

Comment:
Quite a big diff between request and limit. What are these values based on?

Reply:
It was from `coredns_deployment.yaml.j2` : 
```
resources:
          limits:
            memory: 170Mi
          requests:
            cpu: 100m
            memory: 70Mi
```
We change the memory from 170 to 500 because we thought it was necessary like in the CI.
I think we can revert the change from 500 to 170 or decrease even further if you want.